lib/transport.js

```diff
@@ -0,0 +1,19 @@
+'use strict'
+
+const { Worker } = require('worker_threads')
+
+function transport (filePath, args = []) {
+  args.unshift(filePath)
+  const code = `
```

Comment:
Do we really need to use inline code generation?

Reply:
I have some code to do this that is not doing live code gen. I’ll stick it into a module asap.

Reply:
If you look at the prior commit, it wasn't doing this.
The only reason I did this was patch `process.stdin.fd` so pino-pretty didn't crash.
I did it this way so it's easy to see what's going on, but can we simply load another file (or the same file and check `isMainThread`) - it's no biggie.